tests/jax/test_praxis_layers.py

```diff
@@ -1007,6 +1009,7 @@ def generate_praxis_p_and_flax_cls(self, dtype, attrs):
                            layernorm_type=layernorm_type,
                            hidden_dropout=hidden_dropout,
                            attention_dropout=attention_dropout,
+                           intermediate_dropout=intermediate_dropout,
```

Comment:
Do you test different value for this new parameter?

Reply:
Not in UT, but will in Pax with 5B models